jvm-libs/generic/logging/src/main/kotlin/linea/logging/TimeMeasureLogger.kt

```diff
@@ -0,0 +1,38 @@
+package linea.logging
+
+import org.apache.logging.log4j.Level
+import org.apache.logging.log4j.Logger
+import kotlin.time.measureTime
+
+/**
+ * This is more for debugging purposes, to measure the time taken by a certain action and log it.
+ * For production code, consider using metrics.
+ */
+fun <T> measureTimeAndLog(
```

Comment:
Since it's not a production code, maybe it's better to move it to `:linea:testing`? Otherwise it will just take up a little place in production jars

Reply:
It's not advised for prod but it was meant to be used everywhere. This is extremely useful wen we want o measure things quickly in prod code, take a decision, and them implement metrics in final solution.